http/oidc/src/main/java/org/wildfly/security/http/oidc/Oidc.java

```diff
@@ -52,10 +52,14 @@ public class Oidc {
     public static final String TEXT_CONTENT_TYPE = "text/*";
     public static final String DISCOVERY_PATH = ".well-known/openid-configuration";
     public static final String KEYCLOAK_REALMS_PATH = "realms/";
+    public static final String KEYSTORE_PASS = "password";
+    public static final String PKCS12_KEYSTORE_TYPE = "PKCS12";
```

Comment:
Same here, this looks like something that's only needed for tests, it shouldn't be in the Oidc constants class.

Reply:
moved to keycloakConfiguration class